radio/src/gui/128x64/model_setup.cpp

```diff
@@ -1000,16 +1000,34 @@ void menuModelSetup(event_t event)
                 if (moduleIdx == INTERNAL_MODULE) {
                   uint8_t moduleType = checkIncDec(event, g_model.moduleData[moduleIdx].type, MODULE_TYPE_NONE, MODULE_TYPE_MAX, EE_MODEL,
                                                    isInternalModuleAvailable);
+  #if defined(RADIO_COMMANDO8)
+                    if (reusableBuffer.moduleSetup.newType!=MODULE_TYPE_NONE)
+                    {
+                      reusableBuffer.moduleSetup.newType=MODULE_TYPE_NONE;
+                      g_model.moduleData[moduleIdx].type = reusableBuffer.moduleSetup.newType;
+                      reusableBuffer.moduleSetup.previousType = reusableBuffer.moduleSetup.newType;
+                      setModuleType(EXTERNAL_MODULE, MODULE_TYPE_NONE);
```

Comment:
As I understand it, you are trying to implement mutually exclusive modules between internal and external, right?
This should be implemented in `isInternalModuleAvailable` and `isExternalModuleAvailable`.

Also, the `define` should be called something like `MUTUALLY_EXCLUSIVE_MODULES`.

Reply:
YES